dashboard/test/lib/policies/inline_answer_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class InlineAnswerTest < ActiveSupport::TestCase
+  setup_all do
+    @teacher = create :teacher
+    @student = create :student
+  end
+
+  test 'visible? returns true in levelbuilder' do
+    Rails.application.config.stubs(:levelbuilder_mode).returns true
+
+    assert Policies::InlineAnswer.visible?(@student, nil)
+    assert Policies::InlineAnswer.visible?(@student, create(:script_level))
+  end
+
+  test 'visible? returns false for non teachers' do
+    assert_not Policies::InlineAnswer.visible?(@student, create(:script_level))
+  end
+
+  test 'visible? returns true for authorized teachers in csp' do
+    create(:plc_user_course_enrollment, plc_course: create(:plc_course), user: @teacher)
+    assert Policies::InlineAnswer.visible?(@teacher, create(:script_level))
+  end
```

Comment:
Feels like there's an opportunity to improve these tests.  For example:

- What does `visible?` return for a teacher that is not an authorized teacher?
- Does this last test really test the "in csp" part of its name?
- Does the special case for script_levels in professional learning courses work?

Reply:
Looking closer at this, this test definitely seems odd for a number of reasons. I've gone through and rewritten the tests to match how I would have written them myself for this feature, and ended up taking this test out